src/functions-and-arrays.js

```diff
@@ -52,14 +174,61 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
+const crabArray = ['crab', 'crab', 'crab', 'crab'];
+
+function uniquifyArray(array){
+  if(array.length === 0){
+    return null
+  }
+  for(let i = 0; i <= array.length; i++) {
```

Comment:
Here, you have everything correct, however, there are simpler methods for you to perform this kind of action. It is a good use of nested `for` loops, but you could also use the `.includes()` method to make it easier to write and understand.
It would be something like this:
```
function uniquifyArray(array) {
  if (array.length === 0) return null;
  let uniqueArray = [];
  for (let i = 0; i < array.length; i++) {
    if (!uniqueArray.includes(array[i])) {
      uniqueArray.push(array[i]);
    }
  }
  return uniqueArray;
}
```